src/index.js

```diff
@@ -5,10 +5,21 @@ import ReactDOM from 'react-dom';
 import App from './App';
 
 const elements = document.getElementsByClassName('rstore-domain-search');
-
 Array.prototype.forEach.call(elements, element => {
+  const text = {
+    placeholder: element.dataset.text_placeholder || "Find your perfect domain name",
+    search: element.dataset.text_search || "Search",
+    available: element.dataset.text_available || "Congrats, your domain is available!",
+    notAvailable: element.dataset.text_not_available || "Sorry that domain is taken",
+    cart: element.dataset.text_cart || "Continue to Cart",
+    select: element.dataset.text_select || "Select",
+    selected: element.dataset.text_selected || "Selected"
+  },
+  baseUrl = element.dataset.base_url||"secureserver.net",
```

Comment:
nitpick: add spaces before and after the `||`

`baseUrl = element.dataset.base_url || "secureserver.net",`